dpnp/backend/include/dpnp_gen_2arg_3type_tbl.hpp

```diff
@@ -103,40 +103,6 @@
 
 #endif
 
-MACRO_2ARG_3TYPES_OP(dpnp_add_c,
-                     input1_elem + input2_elem,
-                     x1 + x2,
-                     MACRO_UNPACK_TYPES(bool, std::int32_t, std::int64_t),
-                     oneapi::mkl::vm::add,
-                     MACRO_UNPACK_TYPES(float,
-                                        double,
-                                        std::complex<float>,
-                                        std::complex<double>))
-
-MACRO_2ARG_3TYPES_OP(dpnp_arctan2_c,
-                     sycl::atan2(input1_elem, input2_elem),
-                     sycl::atan2(x1, x2),
-                     MACRO_UNPACK_TYPES(float, double),
-                     oneapi::mkl::vm::atan2,
-                     MACRO_UNPACK_TYPES(float, double))
-
-MACRO_2ARG_3TYPES_OP(dpnp_copysign_c,
-                     sycl::copysign(input1_elem, input2_elem),
-                     sycl::copysign(x1, x2),
-                     MACRO_UNPACK_TYPES(float, double),
-                     oneapi::mkl::vm::copysign,
-                     MACRO_UNPACK_TYPES(float, double))
-
-MACRO_2ARG_3TYPES_OP(dpnp_divide_c,
-                     input1_elem / input2_elem,
-                     x1 / x2,
-                     MACRO_UNPACK_TYPES(bool, std::int32_t, std::int64_t),
-                     oneapi::mkl::vm::div,
-                     MACRO_UNPACK_TYPES(float,
-                                        double,
-                                        std::complex<float>,
-                                        std::complex<double>))
-
 MACRO_2ARG_3TYPES_OP(
     dpnp_fmod_c,
```

Comment:
`dpnp_fmod_c` is not used anymore in dpnp and can be safely remove I believe

Reply:
Will be removed in a separate PR